patches/server/0899-Fix-a-bunch-of-vanilla-bugs.patch

```diff
@@ -44,6 +44,14 @@ https://bugs.mojang.com/browse/MC-258535
   by: Jake Potrebic <jake.m.potrebic@gmail.com>
   Fixes certain explosion damage not scaling with difficulty
 
+https://bugs.mojang.com/browse/MC-210802
+  by: BillyGalbreath <Blake.Galbreath@Gmail.com>
```

Comment:
Also, diff capitalization on emails?

Reply:
Just copied them right from the original patch files.